ContactList/ContactList/Validation.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Validation.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/10/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import Foundation
+
+
+
+func isValidEmail(email: String) -> Bool {
```

Comment:
1. Global functions are evil. Wrap the into some class as static functions
2. Lot of copy-paste. Declare enumeration for validation cases, return regEx as calculated property.